sceptre/cli/init.py

```diff
@@ -48,9 +49,9 @@ def init_project(ctx, project_name):
     Creates PROJECT_NAME project folder and a config.yaml with any
     required properties.
     """
-    cwd = os.getcwd()
-    sceptre_folders = {"config", "templates"}
-    project_folder = os.path.join(cwd, project_name)
+    context = SceptreContext(project_path=ctx.obj["project_path"])
```

Comment:
Same as above, this requires 2 args now.

Reply:
removed altogether this func doesn't require context